app/services/state_file/reminder_to_finish_state_return_service.rb

```diff
@@ -3,15 +3,13 @@ class ReminderToFinishStateReturnService
     def self.run
       cutoff_time_ago = 12.hours.ago
       batch_size = 10
-      intakes_with_no_submission = StateFileAzIntake.where("df_data_imported_at < ?", cutoff_time_ago)
-                                                    .left_joins(:efile_submissions)
-                                                    .where(efile_submissions: { id: nil })
-                                                    .where.not("state_file_az_intakes.message_tracker #> '{messages.state_file.finish_return}' IS NOT NULL")
+      intakes_with_no_submission = StateFileAzIntake.where('state_file_az_intakes.created_at < ?', cutoff_time_ago)
+                                                    .where.not(email_address: nil).where.not(email_address_verified_at: nil)
```

Comment:
do we also need to check `.where(unsubscribed_from_email: false)` ? 
asking since I saw it on the other PR #4475

Reply:
Good call, I will add.